### PR TITLE
Boavizta Pipeline upgrade & Mocks Implementation

### PR DESCRIPTION
**Pull request by gnanakeethan**
Signed-off-by: Gnanakeethan Balasubramaniam <gnanakeethan@users.noreply.github.com>### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### Model Pipeline Upgrade

This PR implements the Model Pipeline workflow for

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,9 +42,7 @@
   },
   "jsii": {
     "versionFormat": "full",
-    "outdir": "./dist",
     "tsc": {
-      "outDir": "./dist",
```
Comment (narekhovhannisyan) [inline package.json:46]: why?

**Review by jawache — APPROVED** [review]
Looks good to me

File: package.json (hunk 1 above)
Comment (gnanakeethan) [reply package.json:46]: otherwise the tests will fail and conflict.

Comment (gnanakeethan) [reply package.json:46]: The build process does not include packaging as of now and we are relying on the JSII compiler. 

I am bringing in some changes that resolve in another PR I think.

**Pull request merged**